pkg/baremetal/remediator/remediator.go

```diff
@@ -78,6 +82,13 @@ func (bmr *BareMetalRemediator) Reboot(ctx context.Context, machineRemediation *
 		// it can mean that an user power off the machine by purpose
 		if !bmh.Spec.Online && !rebootInProgress {
 			glog.V(4).Infof("Skip the remediation, machine %q has power off state before the remediation action", machine.Name)
+			bmr.recorder.Eventf(
+				machine,
+				corev1.EventTypeNormal,
+				"MachineRemediationSkippedOffline",
```

Comment:
can you please create consts under for each event type, similar to https://github.com/kubevirt/machine-remediation-operator/blob/826d7775c2fb1673dbfe4653113c66ba152a4bc1/pkg/apis/machineremediation/v1alpha1/machineremediation_types.go#L20